ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/ClosedCases.cshtml.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace ConcernsCaseWork.Pages.Trust;
+
+public class ClosedCases : PageModel
```

Comment:
QUESTION:

Are these files meant to be in this review?

Reply:
removed